zod/src/zod.ts

```diff
@@ -64,22 +64,22 @@ export const zodResolver: Resolver =
       options.shouldUseNativeValidation && validateFieldsNatively({}, options);
 
       return {
-        errors: {},
+        errors: {} as FieldErrors<any>,
```

Comment:
wondering why have to start casting now.

Reply:
I don't know exactly but seems like an empty object is not assignable to the DeepMap with the new version of FieldErrors from react-hook-form?